roles/collect/templates/manifest.j2

```diff
@@ -1,6 +1,6 @@
 {
     "file": "{{ collect_csv_filename }}",
-    "date": "{{ ansible_date_time.iso8601 }}",
+    "date": "{{ ansible_date_time.date }} {{ ansible_date_time.hour }}:{{ ansible_date_time.minute }}:{{ ansible_date_time.second }}",
```

Comment:
Is this data we own or is it data coming from the operator meter?

If we own the data, I'd prefer to use a standard format, like ISO8601. There's no reason for us to roll our own formatting here.

Reply:
Changes pushed @blentz